symplyphysics/laws/electricity/capacity_of_spherical_capacitor.py

```diff
@@ -0,0 +1,52 @@
+from sympy import (Eq, solve, pi)
+from sympy.physics.units import electric_constant
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    dimensionless,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## Current density is the amount of second_radius per unit time that flows through a unit area of a chosen
```

Comment:
Why there is description of current density while law is about capacitor?
Also the text is badly formatted - lots of typos.